simple/stats/db.py

```diff
@@ -337,9 +367,22 @@ def __init__(self, db_params: dict) -> None:
     logging.info("Connected to SQLite: %s", self.local_db_file_path)
 
     self.cursor = self.connection.cursor()
+    # Drop indexes first so inserts are faster.
+    self._drop_indexes()
```

Comment:
Im not sure it makes sense to do drop indexes in the constructor. i might not be fully understanding, but while i think it works for our current workflow, i could see situations where we'd want to create this object to just run read operations?

Reply:
Keeping the index slows down inserts. So dropping and then readding helps with throughput and ensures the tables are indexed at the end.